### PR TITLE
Rethink the README walk-through

### PR DESCRIPTION
**Pull request by lordkada**
#### List of Changes
Changed the README walk-through.

#### Motivation and Context
As a developer, I want to have a simple README file that walks me through the most common scenarios, i.e. running from the source code (yes, I'm on GitHub) or a pre-built docker image (for a super-fast and easy go!). This means also that everything else just falls into more advanced scenarios (such as building a docker image locally).

Starting from this simple consideration, the structure of the README profoundly simplifies leading to this proposed PR.

#### Types of changes

- [X] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,79 +38,85 @@ Create a `NotificationRequest` providing two documents: the act to notify and th
...
 ```
 
-### Run using Docker (Dockerfile)
+### Run as a Docker image
 
-The repository comes with a Dockerfile that you can use to run the application with Docker.
+Another option is to run the container image already available in the containers registry, as follows:
 
-Build the image.
+1. Pull the image from the container registry.
```
Comment (datalek) [inline README.md:92]: The pull is optional, so we can remove it. The command `docker run -p 3000:3000 ghcr.io/pagopa/pn-local-emulator:latest` does the pull if needed

**Review by datalek — APPROVED** [review]
👌

File: README.md (hunk 1 above)
Comment (lordkada) [reply README.md:92]: Agreed. Simpler is better

**Review by datalek — APPROVED** [review]
LGTM 💥

**Pull request merged**